core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -592,13 +600,13 @@ class ReplicaManager(val config: KafkaConfig,
                  _: ReplicaNotAvailableException |
                  _: OffsetOutOfRangeException) =>
           LogReadResult(FetchDataInfo(LogOffsetMetadata.UnknownOffsetMetadata, MemoryRecords.EMPTY), -1L,
-            partitionFetchSize, false, Some(e))
+            -1L, -1L, partitionFetchSize, Some(e))
```

Comment:
Sure. All instantiation of LogReadResult are using named params now.

I didn't change this because most values are -1L already.